docs/user_guide/getting_started.rst

```diff
@@ -28,6 +28,27 @@ installed. There are two options for this:
 Install boa
 ===========
 
+If you need to create a new environment:
+
+     conda create -n boa
+     conda activate boa
+
+If you don't need to create a new environment, activate that environment
+Then install dependencies
+
+    conda install botorch -c pytorch -c gpytorch -c conda-forge
+    pip install ax-platform
+
+Install BOA
+    pip install git+https://github.com/madeline-scyphers/boa.git
+
+Install the develop version of BOA
+    pip install git+https://github.com/madeline-scyphers/boa.git@develop
```

Comment:
There are a couple of issues with rst formatting for the code blocks. I'm pushing a commit for these to be sure they render properly